toolchains/sysimage/build_container_base_image.py

```diff
@@ -93,23 +93,34 @@ def main():
     args = parser.parse_args()
 
     log.info(f"Using args: {args}")
-    temp_sys_dir = process_temp_sys_dir_args(args.fancy.temp_container_sys_dir, args.fancy.tmpfs_container_sys_dir)
 
     build_args = list(args.fancy.build_args or [])
-    context_dir = os.getenv("ICOS_TMPDIR")
-    if not context_dir:
-        raise RuntimeError("ICOS_TMPDIR env variable not available, should be set in BUILD script.")
+    context_dir = tempfile.mkdtemp()
 
     # Add all context files directly into dir
     for context_file in args.context_files:
         shutil.copy(context_file, context_dir)
 
-    container_cmd = generate_container_command("sudo podman ", temp_sys_dir)
+    # Temporary shim to create tmpfs on demand, until we have userspace
+    # overlayfs, or tmpfs natively available on CI.
```

Comment:
We can't run overlayfs on overlayfs, so keep this shimmed until we switch to a common tmpfs.

This is not an issue for local builds, as we already bind `/tmp`.